cmd/crossrefindexer/main.go

```diff
@@ -44,6 +55,7 @@ func main() {
 		if !open {
 			break
 		}
-		log.Println(pub.Doi)
+		test := crossrefindexer.BuildBibliographicField(&pub)
```

Comment:
I think you should create a function with the following signature instead: `func toSimplifiedPublication(pub Crossref) SimplifiedPublication`

Reply:
Also, it needs tests